notebooks/wps_evaluate_rules_demo.ipynb

```diff
@@ -33,7 +33,7 @@
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "Using sandpiper on https://docker-dev03.pcic.uvic.ca/twitcher/ows/proxy/sandpiper/wps\n"
+      "Using sandpiper on https://marble-dev01.pcic.uvic.ca/twitcher/ows/proxy/sandpiper/wps\n"
```

Comment:
There are a lot of hardcoded server specs in this bird. I wonder if they can be factored out into ENV vars or the like, so that they are shared and easily updated.